# always run /usr/games/flare because it has to change the path
# so the game finds its data files
Index: flare-0.19/distribution/flare.desktop.in
===================================================================
--- flare-0.19.orig/distribution/flare.desktop.in	2013-12-02 23:47:07.000000000 +0100
+++ flare-0.19/distribution/flare.desktop.in	2014-08-03 15:02:42.818994571 +0200
@@ -14,8 +14,8 @@
 Comment[gl]=Xogo individual de acción e rol cunha vista isométrica en dúas dimensións.
 Comment[ru]=Изометрический ролевой боевик для одного игрока
 Comment[uk]=Ізометричний рольовий екшн (Action/RPG) для одного гравця
-TryExec=@FLARE_EXECUTABLE_PATH@ --game=flare-game
-Exec=@FLARE_EXECUTABLE_PATH@ --game=flare-game
+TryExec=/usr/games/flare --game=flare-game
+Exec=/usr/games/flare --game=flare-game
 Categories=Game;RolePlaying;
 Icon=flare
 Terminal=false
